### DISCUSSION TITLE
Sourcing and Updates via Package Manager

### DISCUSSION THREAD

**Discussion by brikabrak** [FirstGearGames/FishNet#204] [Feature Request]
While looking over your project, I noted your update and 'no-break' ethos and think it would fit well into the versioning system that Unity's package manager uses. It would be cool to get this asset and its version updates through that instead of requiring asset source inside our own projects. I also know that several open-source packages use [OpenUPM](https://openupm.com/docs/) as a distribution platform.

**Comment by FirstGearGames** [comment]
This requires setting up the project and git in a different manner. But also, users require git to be installed to use what you are requesting.
While I understand the versioning system approach would be super handy for some, it would also be very problematic for others.

**Reply by brikabrak** [reply]
Ah that's a shame. I'd have enjoyed keeping it separate overall from our main project's source. Thanks anyway.
